<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStu</class>
 <widget class="QDialog" name="addStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/png/小17.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Name">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Passwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>不少于6位</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Passwds">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,2,0">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_Male">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>男</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_Female">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,3,10">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_Age">
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>31</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>33</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>34</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>35</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>37</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>39</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>41</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>42</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>43</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>44</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>46</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>47</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>49</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>51</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>53</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>54</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>56</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>58</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>59</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>61</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>62</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>63</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>65</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>66</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>67</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>68</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>69</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>70</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,8,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>密保问题：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Que"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,8,0">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="text">
        <string>答案：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Ans"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Ok">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cancel">
       <property name="font">
        <font>
         <family>黑体</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Name</tabstop>
  <tabstop>le_Passwd</tabstop>
  <tabstop>le_Passwds</tabstop>
  <tabstop>rbtn_Male</tabstop>
  <tabstop>rbtn_Female</tabstop>
  <tabstop>cbb_Age</tabstop>
  <tabstop>le_Que</tabstop>
  <tabstop>le_Ans</tabstop>
  <tabstop>btn_Ok</tabstop>
  <tabstop>btn_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
